README.md

```diff
@@ -36,3 +36,16 @@ A link to the google calendar item can be found [here](https://calendar.google.c
 We use https://hackmd.io/ for taking meeting minutes and will upload the resultant markdown file after the meeting has concluded.
 
 There is a template provided in [`misc/DEV_MEETING_TEMPLATE.md`](https://github.com/conda-forge/conda-forge.github.io/tree/main/misc/DEV_MEETING_TEMPLATE.md) that you should use to create a new hackmd document.
+
+## Code Of Conduct
+All are welcome to participate in conda-forge community (including the dev meetings) as long as the tenets of this Code of Conduct are adhered to:
+* Treat one another with dignity and respect.
+* Respect and treat others fairly, regardless of race ancestry, place of origin, colour, ethnic origin, citizenship, religion, gender, sexual orientation, age or disability.
```

Comment:
```suggestion
* Respect and treat others fairly, regardless of race, skin color, ethnicity, ancestry, place of origin, national origin, citizenship, religion, gender, sexual orientation, age, or disability.
```

Probably you should get the appropriate list from the reference source (numfocus) or follow some standard definition (e.g. in the US, there is a law about "protected characteristics" and most of these fall under that)